imap_processing/tests/swe/test_swe_l1b.py

```diff
@@ -69,11 +69,11 @@ def test_swe_l1b(decom_test_data):
 def test_cdf_creation():
     """Test that CDF file is created and has the correct name."""
     test_data_path = "tests/swe/l0_data/2024051010_SWE_SCIENCE_packet.bin"
-    l1a_datasets = swe_l1a(imap_module_directory / test_data_path)
+    l1a_datasets = swe_l1a(imap_module_directory / test_data_path, "v002")
 
     sci_l1a_filepath = write_cdf(l1a_datasets)
 
-    assert sci_l1a_filepath.name == "imap_swe_l1a_sci_20240510_v001.cdf"
+    assert sci_l1a_filepath.name == "imap_swe_l1a_sci_20240510_v002.cdf"
```

Comment:
This is `v002`, but the l1b below this is still `v001`, it looks like the version is set to `None` by default so I'm wondering where the `v001` is getting picked up from. I think this is related to @subagonsouth's comment about putting in a `v999` version attribute or something else to make it obvious we aren't producing the default data version.

Reply:
I did not update swe_l1b to retrieve the data version from the method call yet, so it's still using the old version.